src/components/charts/LineChart.tsx

```diff
@@ -0,0 +1,185 @@
+import React from "react"
+import merge from "deepmerge"
+
+import {
+  ChartWrapper,
+  RecordSet,
+  RecordEntry,
+  ChartProps,
+} from "./ChartWrapper"
+
+export interface LineChartProps extends ChartProps {
+  strokeWidths?: number | number[]
+  /**
+   * The type of dashed lines for each line. Higher numnbers means more space between dashes.
+   */
+  dashTypes?: number | number[]
+  /**
+   * Whether each line is curved.
+   */
+  curved?: boolean
+  /**
+   * Specify the size of markers for each line. 0 means no markers.
+   */
+  markerSizes?: number | number[]
+  /**
+   * The title of the chart
+   */
+  xMin?: number
+  /**
+   * The maximum value of the x (horizontal) or y axis. Useful when data labels overflow.
+   */
+  xMax?: number
+  /**
+   * The minimum value of the x (horizontal) or y axis.
+   */
+  yMin?: number
+  /**
+   * The maximum value of the x (horizontal) or y axis. Useful when data labels overflow.
+   */
+  yMax?: number
+  /**
+   * The width of the chart object. Defaults to 100% so it is responsive.
+   */
+}
+
+interface DataPoint {
```

Comment:
I am a wee bit surprised that these interfaces aren't in the base ApexCharts.... Is the issue that ApexCharts is pure JS and so doesn't export these types?

Reply:
Seems to me ApexCharts is pure JS. They did add a Type declaration but only exported `ApexOptions` for their options interface. Other types are private but does have type check